libevm/ExtVMFace.h

```diff
@@ -169,6 +169,33 @@ class EnvInfo
     u256 m_gasUsed;
 };
 
+/// Represents a call result.
+///
+/// @todo: Replace with evmc_result in future.
+struct CallResult
+{
+    evmc_status_code status = EVMC_INTERNAL_ERROR;
```

Comment:
Also no real need to use inline initialization here as it's always initialized by a single constuctor

Reply:
I prefer to have every field initialized, but in this case I can skip it to avoid confusion.